modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/utils/sortItems.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {IOrderable} from './types';
+
+export default function sortItems(
+	items: IOrderable[],
+	itemsOrder: string
+): IOrderable[] {
+
+	const itemsOrderArray = itemsOrder?.split(',') || [] as string[];
+
+	let ordered: IOrderable[] = [];
+	let notOrdered: IOrderable[] = [];
+
+	ordered = itemsOrderArray
+		.map((itemId) => items.find((item) => item.id === Number(itemId)))
+		.filter(Boolean) as IOrderable[];
+
+	// TODO: really sort by date
+	notOrdered = items
+		.filter((item) => !itemsOrderArray.includes(String(item.id)))
+		.sort((item1, item2) =>
+			item1.dateCreated < item2.dateCreated
+				? 1
+				: item1.dateCreated > item2.dateCreated
+				? -1
+				: 0
+		);
```

Comment:
I don't think this sorting is needed here if we include `&sort=dateCreated:desc` in the request like the one in the sort tab includes

https://github.com/liferay-frontend/liferay-portal/blob/5a55d3c7ac187514f3eea6d3ec4c24e2c2e681d2/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Sorting.tsx#L451

Reply:
Mmm that makes sense, if we are sure we request all the items in that way, then the ones not affected by the `order` field would automatically be sorted. I'll try this!